lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -48,15 +48,25 @@ function Install-Project {
     $daemon = 'auto',
     [string]
     $http_proxy,
-    [string]
-    $download_url_override,
     # Specify an alternate download url, must also include checksum
     [string]
-    $checksum
+    $download_url_override,
     # SHA256 checksum of the download file
     # Must be present when using download_url_override
+    [string]
+    $checksum,
+    # Set to 'once' to skip install if project is detected
+    [string]
+    $install_strategy
   )
 
+  if ((Test-Path "$env:systemdrive\opscode\$project\embedded") -and ($install_strategy -eq 'once')) {
```

Comment:
Is there a reason for the mismatch on which path is checked on bash vs. powershell (root vs root/embedded)?